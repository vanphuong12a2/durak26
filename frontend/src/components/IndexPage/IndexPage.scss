header {
  text-align: center;
  margin: auto;
  max-width: 600px;
}

.new-game {
  text-align: center;
}
