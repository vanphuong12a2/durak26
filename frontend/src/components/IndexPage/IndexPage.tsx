import React from 'react';

const IndexPage = () => {
  return (
    <div>Index page</div>
  );
};

export default IndexPage;
