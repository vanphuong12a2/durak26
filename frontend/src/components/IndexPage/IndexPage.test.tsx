import React from 'react';
import {create} from 'react-test-renderer';
import IndexPage from './IndexPage';


describe('<IndexPage />', () => {

  describe('Snapshots', () => {
    it('renders correctly', () => {
      const app = create(<IndexPage/>);
      expect(app.toJSON()).toMatchSnapshot();
    });
  });

});
