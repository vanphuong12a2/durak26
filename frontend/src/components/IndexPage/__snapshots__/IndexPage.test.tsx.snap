// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<IndexPage /> Snapshots renders correctly 1`] = `
<div>
  Index page
</div>
`;
